.travis.yml

```diff
@@ -5,9 +5,7 @@ sudo: false
 
 cache: pip
 
-install:
-  - pip install -r requirements.txt
```

Comment:
Does this mean that requirements.txt can be removed?

Reply:
:+1: 4ab30a8